csharp/src/Ice/Runtime.cs

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+
+namespace ZeroC.Ice
+{
+    public static class Runtime
+    {
+        /// <summary>The process-wide logger.</summary>
+        public static ILogger Logger
+        {
+            get
+            {
+                lock (_processLoggerMutex)
+                {
+                    if (_processLogger == null)
+                    {
+                        _processLogger = new ConsoleLogger(AppDomain.CurrentDomain.FriendlyName);
+                    }
+                    return _processLogger;
+                }
+            }
+            set
+            {
+                lock (_processLoggerMutex)
+                {
+                    _processLogger = value;
+                }
+            }
+        }
+
+        /// <summary>Returns the Ice version as an integer in the form A.BB.CC, where A indicates the major version,
+        /// BB indicates the minor version, and CC indicates the patch level. For example, for Ice 3.3.1, the returned
+        /// value is 30301.</summary>
+        /// <returns>The Ice version.</returns>
+        public static int IntVersion => 40000; // AABBCC, with AA=major, BB=minor, CC=patch
```

Comment:
Both IntVersion and StringVersion should be const.

Reply:
Fixed